### DISCUSSION TITLE
Serving the Lit 3.0 my-element demo from the NPM CDN (unpkg)?

### DISCUSSION THREAD

**Discussion by fireflysemantics** [lit/lit#4664] [Q&A]
Any ideas?

https://stackoverflow.com/questions/78577676/serving-the-lit-3-0-my-element-demo-from-the-npm-cdn-unpkg

**Comment by justinfagnani** [comment]
A comment on this discussion thread: there isn't much info to go on here without clicking through links. That's not really easy for people reading who might be able to help. I would describe your problem and question here if you want to entice more people to help.
